wrappers/leveldb/leveldb_wrapper.h

```diff
@@ -138,3 +150,48 @@ int leveldb_wrapper::scan(const char* key, size_t key_sz, int scan_sz, char*& va
 	// FIXME(jhpark) : the return type of "scan" funcction should be changed as boolean.
 	return 1;
 }
+
+void leveldb_wrapper::print_stat(leveldb::DB* db_, bool print_sst) {
+
+	std::string stats_property;
+	std::string sst_property;
+	std::string mem_usage_property;
+	std::string numfiles_level_property;
+
+	std::cout << std::string(50, '=') << "\n";
+	if(print_sst) {
+		db_->GetProperty(LEVELDB_SSTTABLE_STATS, &sst_property);
+		std::cout << "LevelDB SST Description:"
+							<< "\n"
+							<< sst_property << std::endl;
+	}
+
+	db_->GetProperty(LEVELDB_STAT, &stats_property);
+	std::cout << "LevelDB Stats:"
+						<< "\n"
+						<< stats_property << std::endl;
+
+	db_->GetProperty(LEVELDB_MEMORY_USAGE, &mem_usage_property);
+	std::cout << "LevelDB Memory Usage:"
+						<< "\n\t"
+						<< mem_usage_property << " (B)" << std::endl;
+
+	// print number of files at each levels
+	std::cout << "LevelDB Number of files at each levels:" << std::endl;
+	uint32_t level = 0;
+	uint32_t total_files = 0;
+	int num_files = 0;
+
+	while(true) {
+		db_->GetProperty(LEVELDB_NUM_FILES_AT_LEVEL + std::to_string(level), &numfiles_level_property);
+		std::istringstream(numfiles_level_property) >> num_files;
+		if (num_files == 0) {
```

Comment:
I see a problem here. It can be that some level is empty but the next levels have files. In this case the loop will break and next levels will not be printed. For example, I got the following output when testing:
```
==================================================
LevelDB Stats:
                               Compactions
Level  Files Size(MB) Time(sec) Read(MB) Write(MB)
--------------------------------------------------
  0        0        0         9        0       588    <-----------
  1       47       89         9      602       597
  2      211      424        13      866       924
  3       64      129         0        0         0

LevelDB Memory Usage:
        54896307 (B)
LevelDB Number of files at each levels:
        Total: 0                                       <-----------
==================================================
```
Looking at LevelDB source code, `GetProperty()` returns false in case you request a level greater than the maximum defined internally by `config::kNumLevels`. So using the return value is probably the best condition to break the loop.
See: https://github.com/google/leveldb/blob/1.22/db/db_impl.cc#L1385